cmd/iam.go

```diff
@@ -522,9 +522,12 @@ func (sys *IAMSys) ListPolicies(ctx context.Context, bucketName string) (map[str
 		return nil, errServerNotInitialized
 	}
 
-	<-sys.configLoaded
-
-	return sys.store.ListPolicies(ctx, bucketName)
+	select {
+	case <-sys.configLoaded:
+		return sys.store.ListPolicies(ctx, bucketName)
+	default:
+		return nil, errIAMInitializeFailed
```

Comment:
Thanks for the PR.

I think you need to use errIAMNotInitialized rather than failed. Currently the initialization of IAM happens in the background so we don't know here if there is something wrong or it is just initializing.